streams/src/main/java/org/apache/kafka/streams/processor/internals/StateDirectory.java

```diff
@@ -45,21 +45,29 @@
     private static final Logger log = LoggerFactory.getLogger(StateDirectory.class);
 
     private final File stateDir;
-    private final String logPrefix;
     private final HashMap<TaskId, FileChannel> channels = new HashMap<>();
-    private final HashMap<TaskId, FileLock> locks = new HashMap<>();
+    private final HashMap<TaskId, LockAndOwner> locks = new HashMap<>();
     private final Time time;
 
     private FileChannel globalStateChannel;
     private FileLock globalStateLock;
 
+    private static class LockAndOwner {
+        final FileLock lock;
+        final long owningThreadId;
+
+        LockAndOwner(long owningThreadId, FileLock lock) {
+            this.owningThreadId = owningThreadId;
+            this.lock = lock;
+        }
+    }
+
```

Comment:
For line 65 and line 69: seems we can collapse them into one constructor as `threadId` is not useful anymore.